File: apps/mobile/ios/Visvine/Models/Community.swift
import Foundation

/// Tenant community — mirrors the web API's `Community` DTO (server truth).
struct Community: Codable, Identifiable, Equatable {
    let id: String
    let name: String
    var description: String?
    var imageUrl: String?
    var image: String?
}

struct CommunitiesResponse: Codable {
    let communities: [Community]
}

File: apps/mobile/android/app/src/main/java/com/visvine/mobile/data/model/Community.kt
package com.visvine.mobile.data.model

import kotlinx.serialization.Serializable

/** Tenant community — mirrors docs/native-migration/api-contract.md `Community`. */
@Serializable
data class Community(
    val id: String,
    val name: String,
    val description: String? = null,
    val imageUrl: String? = null,
    val image: String? = null,
)

@Serializable
data class CommunitiesResponse(
    val communities: List<Community> = emptyList(),
)
